Classes/Core/Functional/FunctionalTestCase.php

```diff
@@ -891,7 +891,12 @@ protected function retrieveFrontendRequestResult(InternalRequest $request, Inter
         ];
 
         $vendorPath = (new Testbase())->getPackagesPath();
-        $template = new \Text_Template($vendorPath . '/typo3/testing-framework/Resources/Core/Functional/Fixtures/Frontend/request.tpl');
```

Comment:
This solve the issue. For clarity and to avoid duplication I'd do it like this though:

```suggestion
        $templateClass = \Text_Template::class;
        if (!class_exists($templateClass)) {
        	$templateClass = \SebastianBergmann\Template\Template::class;
        }
        $template = new $templateClass($vendorPath . '/typo3/testing-framework/Resources/Core/Functional/Fixtures/Frontend/request.tpl');
```